### PR TITLE
feat(blur): terminate blur web worker when disabled

### PR DESCRIPTION
**Pull request by tflanagan**
Resolve a potential leak. Terminate the blur web worker when the effect is disabled.

### CODE REVIEW

**Comment by tflanagan** [comment]
I'm unsure how to overcome these lint errors. Any suggestions?

File: react/features/stream-effects/blur/JitsiStreamBlurEffect.js [hunk 1]
```diff
@@ -130,8 +132,14 @@ export default class JitsiStreamBlurEffect {
...
     stopEffect() {
-        this._maskFrameTimerWorker.postMessage({
-            id: CLEAR_INTERVAL
-        });
+        if (this._maskFrameTimerWorker !== undefined) {
+            this._maskFrameTimerWorker.postMessage({
+                id: CLEAR_INTERVAL
+            });
+
+            this._maskFrameTimerWorker.terminate();
```
Comment (saghul) [inline react/features/stream-effects/blur/JitsiStreamBlurEffect.js:140]:
```suggestion
            this._maskFrameTimerWorker?.terminate();
```

File: react/features/stream-effects/blur/JitsiStreamBlurEffect.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default class JitsiStreamBlurEffect {
     _bpModel: Object;
     _inputVideoElement: HTMLVideoElement;
     _onMaskFrameTimer: Function;
-    _maskFrameTimerWorker: Worker;
+    _maskFrameTimerWorker: Worker | typeof undefined;
```
Comment (saghul) [inline react/features/stream-effects/blur/JitsiStreamBlurEffect.js:21]:
```suggestion
    _maskFrameTimerWorker: ?Worker;
```

**Comment by tflanagan** [comment]
@saghul 

Thanks for the suggestion, but it doesn't appear to be working.

I suppose there isn't too much danger to not setting it as undefined and checking if it is defined before use. From the code paths, it doesn't appear that this could happen.

Thoughts?

**Comment by jallamsetty1** [comment]
> @saghul
> 
> Thanks for the suggestion, but it doesn't appear to be working.
> 
> I suppose there isn't too much danger to not setting it as undefined and checking if it is defined before use. From the code paths, it doesn't appear that this could happen.
> 
> Thoughts?

Yes, that should be fine.

**Comment by jallamsetty1** [comment]
Jenkins please test this please.

**Pull request merged**